### ISSUE TITLE
[themes] Error highlighting theme for colorblind.  

### ISSUE THREAD

**Issue by sushant4032** [microsoft/vscode#53977]
Current error highlighting schemes are very difficult to spot for colorblind people (red-green). When the error is on a single character it is impossible to spot. Errors and warnings are difficult to distinguish. For errors and warnings entire line should have a highlighted background.

**Comment by aeschli** [comment]
Note that you can customize the colors or the error, warning, and info squiggly.
If you set the border color you will additionally get an underline.
```    
"workbench.colorCustomizations": {
        "editorError.border": "#ff0000",
        "editorError.foreground": "#ff0000",
        "editorWarning.border": "#ffff00",
        "editorWarning.foreground": "#ffff00",
        "editorInfo.border": "#00ff00",
        "editorInfo.foreground": "#00ff00",
    }
```

**Comment by Susi2304** [comment]
This helps to some extent, but can the vs-code not have a colorblind friendly mode like many websites do.

**Comment by ouchxp** [comment]
Agree, have been suffer from this issue for a long time. VSCode gives double line style border for `editorError.border` attribute which is really strange looking. It would be the best if the border and background style can be customized too.

**Comment by ianizaguirre** [comment]
I found a way to change the error red squiggly line into an outline style (kind of like Sublime Text), so errors are much easier to spot! I like outlined errors MUCH better over small squiggly lines, because I find the squiggly lines to be difficult to see when scrolling through a file. 

So this was my end result in VS Code:

![vscode-squiggly-error-to-outline-error](https://user-images.githubusercontent.com/29879270/48297096-a3c60800-e46e-11e8-8593-7b2b5b4a0a43.png)

I wrote down the steps I followed to get this to work [HERE](https://www.izaguir.re/how-to-make-vscode-error-highlighting-look-more-like-sublime-error-highlighting/)

Hopefully this solution is helpful 😄

**Comment by ouchxp** [comment]
@ianizaguirre Have tried this when I started to use VSCode, however the extension will be disabled everytime there is an update. And I see no reason VSCode use this error style when I have custom border color 
```    "workbench.colorCustomizations": {
        "editorError.foreground": "#000000",
        "editorError.border": "#FF4D59",
    },
```
<img width="418" alt="image" src="https://user-images.githubusercontent.com/1253729/49979970-4580cd80-ffb6-11e8-8c9e-165a235ec305.png">

**Comment by isidorn** [comment]
We now started using Red - Yellow. So please try that with vscode insiders https://code.visualstudio.com/insiders/
And please provide feedback, @misolori would be very interested

You can also customise the colors as @aeschli metnioned
For decorating the whole line, duplicate of https://github.com/microsoft/vscode/issues/16222

Due to the above, closing this issue
